util/customTransformer.js

```diff
@@ -275,4 +285,22 @@ async function userTransformHandler(
   return events;
 }
 
-exports.userTransformHandler = userTransformHandler;
+async function setupUserTransformHandle(
```

Comment:
do you mean setupUserTransformHandler ?
If yes then we need to make sure that we are in sync with AWS Lambda terminology

Reply:
terminology is related to existing transformer naming